test/Interop/Cxx/static/static-var-irgen.swift

```diff
@@ -0,0 +1,90 @@
+// CHECK: @{{_ZL19staticConstexprInst|staticConstexprInst}} = internal constant %class.NonTrivial { i32 8192 }, align 4
+
+// CHECK: define internal void @{{__cxx_global_var_init|"\?\?__EstaticVarInit@@YAXXZ"}}()
+// CHECK: %call = call i32 @{{_Z13makeStaticVarv|"\?makeStaticVar@@YAHXZ"}}()
+// CHECK: store i32 %call, i32* @{{_ZL13staticVarInit|staticVarInit}}, align 4
+
+// CHECK: declare{{( dso_local)?}} i32 @{{_Z13makeStaticVarv|"\?makeStaticVar@@YAHXZ"}}()
+
+// CHECK: define internal void @{{__cxx_global_var_init.1|"\?\?__EstaticVarInlineInit@@YAXXZ"}}()
+// CHECK: %call = call i32 @{{_Z19inlineMakeStaticVarv|"\?inlineMakeStaticVar@@YAHXZ"}}()
+// CHECK: store i32 %call, i32* @{{_ZL19staticVarInlineInit|staticVarInlineInit}}, align 4
+
+// CHECK: define linkonce_odr{{( dso_local)?}} i32 @{{_Z19inlineMakeStaticVarv|"\?inlineMakeStaticVar@@YAHXZ"}}()
+// CHECK: ret i32 8
+
+// CHECK: define internal void @{{__cxx_global_var_init.2|"\?\?__EstaticConstInit@@YAXXZ"}}()
+// CHECK: %call = call i32 @{{_Z15makeStaticConstv|"\?makeStaticConst@@YAHXZ"}}()
+// CHECK: store i32 %call, i32* @{{_ZL15staticConstInit|staticConstInit}}, align 4
+
+// CHECK: declare{{( dso_local)?}} i32 @{{_Z15makeStaticConstv|"\?makeStaticConst@@YAHXZ"}}()
+
+// CHECK: define internal void @{{__cxx_global_var_init.3|"\?\?__EstaticConstInlineInit@@YAXXZ"}}()
+// CHECK: %call = call i32 @{{_Z21inlineMakeStaticConstv|"\?inlineMakeStaticConst@@YAHXZ"}}()
+// CHECK: store i32 %call, i32* @{{_ZL21staticConstInlineInit|staticConstInlineInit}}, align 4
+
+// CHECK: define linkonce_odr{{( dso_local)?}} i32 @{{_Z21inlineMakeStaticConstv|"\?inlineMakeStaticConst@@YAHXZ"}}()
+// CHECK: ret i32 16
+
+// CHECK: define internal void @{{__cxx_global_var_init.4|"\?\?__EstaticInst@@YAXXZ"}}()
+// CHECK: call {{void @_ZN10NonTrivialC[12]Ei\(%class.NonTrivial\* @_ZL10staticInst, i32 1024\)|%class.NonTrivial\* @"\?\?0NonTrivial@@QEAA@H@Z"\(%class.NonTrivial\* @staticInst, i32 1024\)}}
+
+// CHECK: define internal void @{{__cxx_global_var_init.5|"\?\?__EstaticConstInst@@YAXXZ"}}()
+// CHECK: call {{void @_ZN10NonTrivialC[12]Ei\(%class.NonTrivial\* @_ZL15staticConstInst, i32 2048\)|%class.NonTrivial\* @"\?\?0NonTrivial@@QEAA@H@Z"\(%class.NonTrivial\* @staticConstInst, i32 2048\)}}
+
+public func readStaticVar() -> CInt {
+  return staticVar
+}
+
+// CHECK: define {{(protected |dllexport )?}}swiftcc i32 @"$s4main13readStaticVars5Int32VyF"()
+// CHECK: [[VALUE:%.*]] = load i32, i32* getelementptr inbounds (%Ts5Int32V, %Ts5Int32V* bitcast (i32* @{{_ZL9staticVar|staticVar}} to %Ts5Int32V*), i32 0, i32 0), align 4
+// CHECK: ret i32 [[VALUE]]
+
+public func writeStaticVar(_ v: CInt) {
+  staticVar = v
+}
+
+// CHECK: define {{(protected |dllexport )?}}swiftcc void @"$s4main14writeStaticVaryys5Int32VF"(i32 %0)
+// CHECK: store i32 %0, i32* getelementptr inbounds (%Ts5Int32V, %Ts5Int32V* bitcast (i32* @{{_ZL9staticVar|staticVar}} to %Ts5Int32V*), i32 0, i32 0), align 4
```

Comment:
Please also add tests for passing the `staticVar` inout.

Reply:
Done.